src/traces/heatmap/calc.js

```diff
@@ -20,6 +18,10 @@ var colorscaleCalc = require('../../components/colorscale/calc');
 var hasColumns = require('./has_columns');
 var convertColumnXYZ = require('./convert_column_xyz');
 var maxRowLength = require('./max_row_length');
+var cleanData = require('./clean_data');
```

Comment:
I would prefer not calling this module `clean_data`.

Trace modules with `#cleanData` methods are called during the `Plotly.plot` pipeline [here](https://github.com/plotly/plotly.js/blob/8265df5acf85ecfe828e25cde97a2a7a5e997694/src/plots/plots.js#L430-L435). At the moment, only `Scatter` has a [`cleanData`](https://github.com/plotly/plotly.js/blob/8265df5acf85ecfe828e25cde97a2a7a5e997694/src/plots/plots.js#L430-L435) method but still adding a `heatmap/clean_data.js` module that has nothing to do with `cleanData` step could potentially be confusing.

I'd vote for `clean_z.js`, but @rreusser I'll let decide on an alternative file name.